src/components/FieldDropdown/index.jsx

```diff
@@ -220,76 +145,72 @@ const FieldDropdown = () => {
   return (
     <div className="Init desktop">
-      {PSA && (
-        <select className="fields" onChange={changePSA} value={field}>
-          <option>examples</option>
-          <optgroup label="PSA">
-            {Object.keys(examples)
-              .filter((site) => examples[site].category === 'PSA')
-              .sort()
-              .map((site) => (
-                <option key={site}>{site}</option>
-              ))}
-          </optgroup>
-          <optgroup label="Resham">
-            {Object.keys(examples)
-              .filter((site) => examples[site].category === 'Resham')
-              .sort()
-              .map((site) => (
-                <option key={site}>{site}</option>
-              ))}
-          </optgroup>
-        </select>
-      )}
-
-      {!PSA && (
-        <select className="fields" onChange={handleDropdown} value={field}>
-          <option value="placeholder">&nbsp;</option>
-          <optgroup label="My fields">
-            {myFields.map((fld, idx) => (
-              // eslint-disable-next-line react/no-unknown-property
-              <option key={idx} checked={fld === field}>
-                {fld.replace('ncalc-', '')}
-              </option>
-            ))}
-          </optgroup>
-          <option disabled>____________________</option>
-
-          <optgroup label="Example data">
-            <option>Example: Grass</option>
-            <option>Example: Legume</option>
-          </optgroup>
-          <option disabled>____________________</option>
+      <PSADropdown
+        label={PSA ? "examples" : ""}
+        items={
+          PSA
+            ? [
+              {
+                label: 'Examples',
+                isHeader: true,
+              },
+              ...Object.keys(examples)
+                .filter((site) => examples[site].category === 'PSA')
+                .sort()
+                .map((site) => ({ value: site, label: site })),
+            ]
+            : [
+              {
+                label: 'My fields',
+                isHeader: true,
+              },
+              { value: '', label: '' },
+              ...myFields.map((fld) => ({
+                value: fld.replace('ncalc-', ''),
+                label: fld.replace('ncalc-', ''),
+              })),
+              {
+                label: 'Example data',
+                isHeader: true,
+              },
+              { value: '', label: '' },
+              { value: 'Example: Grass', label: 'Example: Grass' },
+              { value: 'Example: Legume', label: 'Example: Legume' },
+              (pathname.includes('output') || myFields.length) && (
+                {
+                  label: 'Utilities',
+                  isHeader: true,
+                },
+                { value: '', label: '' },
+                { value: 'Download data', label: 'Download data' }),
+              ...(myFields.length ? [{ value: 'Clear previous runs', label: 'Clear previous runs' }] : []),
+            ]
```

Comment:
Can you take a look at the last item? I'm getting warning in the console :
```
index.jsx:148 Warning: Failed prop type: Invalid prop `items[6]` of type `number` supplied to `PSADropdown`, expected `object`.
```
Also it should show the Utilities header on the last page but it's not showing.